java/src/test/android/app/src/androidTest/java/ai/onnxruntime/example/javavalidater/SimpleTest.java

```diff
@@ -0,0 +1,71 @@
+package ai.onnxruntime.example.javavalidater;
+
+import ai.onnxruntime.*;
```

Comment:
if we import ai.onnxruntime.*, would that cover other imports like ai.onnxruntime.OnnxTensor?
that said, if it's not inconvenient it might be preferable to explicitly import the names we require instead of everything.

Reply:
Yes, it will cover everything including ai.onnxruntime.OnnxTensor. but thanks for the suggestions, since we basically only need onnxtensor, session, etc. here, I will explicitly import them.